tetris.py

```diff
@@ -123,25 +123,33 @@ def new_block(self):
             self.current_block_shape = random.choice(shape_names)
         else:
             self.current_block_shape = self.upcoming_block_shape
-        self.current_block_center = (int(game_width / 2), -2)
+        self.current_block_center = (int(game_width / 2), 8)
         self.upcoming_block_shape = random.choice(shape_names)
+        self.index = 0
 
     def get_current_block(self):
         (x, y) = self.current_block_center
         if self.current_block_shape == "I":
-            return [(x - 2, y), (x - 1, y), (x, y), (x + 1, y)]
+            I = [[(x - 2, y), (x - 1, y), (x, y), (x + 1, y)], [(x, y-3), (x, y-2), (x, y-1), (x, y)]]
+            return I[self.index % len(I)]
         if self.current_block_shape == "L":
-            return [(x - 1, y - 1), (x - 1, y), (x, y), (x + 1, y)]
+            L = [[(x-1, y), (x,y), (x+1, y), (x+1, y-1)], [(x,y), (x-1,y), (x-1,y-1), (x-1,y-2)], [(x, y), (x, y-1), (x+1, y-1), (x+2, y-1)], [(x, y-1), (x+1, y-1), (x+1, y), (x+1,y+1)]]
```

Comment:
If you find these long lines difficult to work with, you can black the code. It will be blacked before merging anyway.

Reply:
blacked